tests/screenshot/apps/components/Dropdown.js

```diff
@@ -0,0 +1,10 @@
+import Dropdown from '../../../../Dropdown';
+import React from 'react';
+
+const DropdownTests = [
+	<Dropdown>{[]}</Dropdown>,
+	<Dropdown direction="up">{[]}</Dropdown>,
+	<Dropdown open>{[]}</Dropdown>
```

Comment:
Could you add some texts for the Dropdowns to show something in screenshot tests?